src/04-modality-specific-files/05-task-events.md

```diff
@@ -110,16 +111,19 @@ sub-control01/
         sub-control01_task-emoface_events.json
 ```
 
-Specific fields related to stimulus presentation details SHOULD also be present in this data dictionary:
+Specific fields related to stimulus presentation details SHOULD also be present
+in this data dictionary:
 
-| Column name                  | Description                                                                                                                                                                                                                      |
-|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| StimulusPresentationOS       | RECOMMENDED. Operating system used to run the stimuli presentation software.                                                                                                                                                     |
-| StimulusPresentationSoftware | RECOMMENDED. Name of the software that was used to present the stimuli ideally with its [Research Resource Identifier](https://scicrunch.org/resources) Examples: 'Psychtoolbox, RRID:SCR_002881' or 'PsychoPy, RRID:SCR_006571' |
-| StimulusPresentationVersion  | RECOMMENDED. Version of the software that was used to present the stimuli.                                                                                                                                                       |
-| StimulusPresentationCode     | RECOMMENDED. URL to code of that was used to present the stimuli.                                                                                                                                                                |
+| Column name                      | Description                                                                                                                                                                                                                                                                                                                                   |
+| -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| StimulusPresentationOS           | RECOMMENDED. Operating system used to run the stimuli presentation software. Should mention the OS type (Windows, MacOS, Linux), distribution (if applicable: Ubuntu, Debian) and the version number or equivalent code name (Windows - 10; MacOS - Catalina or MacOS - 10.15; Linux - Ubuntu - Bionic Beaver or Linux - Ubuntu -18.04.5 LTS) |
```

Comment:
Thanks! I like this level of detail and agree with you that this is a slippery slope :confused: 

However, I would pull these examples out of the table. So that in the table there is a `(for formatting recommendations, see examples below)`.

And then right below the table, there are the examples

Reply:
OS descriptions are now below the table. :smile: